### PR TITLE
[Update] Bundle Haber 

### PR DESCRIPTION
**Pull request by eren885**
image

### CODE REVIEW

File: websites/D/Discord Bot List/dist/metadata.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
     "tr": "Discord Bot List bir bot listesidir."
   },
   "url": "discordbotlist.com",
-  "version": "1.2.4",
+  "version": "1.2.5",
   "logo": "https://imgur.com/JDKasKl.png",
-  "thumbnail": "https://i.imgur.com/p7BC1B5.png",
+  "thumbnail": "https://imgur.com/XPHbc5O.png",
```
Comment (Bas950) [inline websites/D/Discord Bot List/dist/metadata.json:14]: this is not the correct link all imgur links must be i.imgur

**Comment by eren885** [comment]
NOW WHAT

**Comment by eren885** [comment]
OK NVM IM DONE ANANISİKEYİM PR

**Comment by frozaum** [comment]
nah

**Pull request merged**